libminifi/include/utils/StringUtils.h

```diff
@@ -74,12 +74,10 @@ struct string_traits<wchar_t>{
 class StringUtils {
  public:
   /**
-   * Converts a string to a boolean
-   * Better handles mixed case.
+   * Checks and converts a String to a Boolean
```

Comment:
why are the types uppercase?

Reply:
I always thought of datatypes as proper nouns lol. I can see that it is wrong. Changing it now.